Tests/Command/RedisFlushAllCommandTest.php

```diff
@@ -33,10 +33,27 @@ public function testWithDefaultClientAndNoInteraction()
             ->method('get')
             ->with($this->equalTo('snc_redis.default'));
 
-        $this->predisClient->expects($this->once())
-            ->method('__call')
-            ->with($this->equalTo('flushall'))
-            ->will($this->returnValue(true));
+        if (!($this->predisClient instanceof \IteratorAggregate)) { // BC for Predis 1.0
+            $this->predisClient->expects($this->once())
+                ->method('__call')
+                ->with($this->equalTo('flushall'))
+                ->will($this->returnValue(true));
+        } else {
+            $node1 = $this->getMockBuilder('\Predis\\Client')->getMock();
```

Comment:
```suggestion
            $node1 = $this->getMockBuilder('\Predis\Client')->getMock();
```

Reply:
Sorry, but i copy/paste this code style from [here](https://github.com/snc/SncRedisBundle/blob/master/Tests/Command/CommandTestCase.php#L53). Your code style is changed?

Reply:
It will be better:
```diff
- $node1 = $this->getMockBuilder('\Predis\\Client')->getMock();
+ $node1 = $this->getMockBuilder(Client::class)->getMock();
```
and
```diff
- $this->equalTo('flushall')
+ self::equalTo('flushall')
```